packages/client/index.html

```diff
@@ -37,72 +37,75 @@
     <title>Loading...</title>
 
     <style>
-      body,
-      html {
+      body {
         background-color: #f5f5f5;
+        margin: 0;
       }
-      #spinner {
+      .progress-container {
+        height: 100vh;
+        width: 100vw;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
         opacity: 0;
-        position: absolute;
-        left: 50%;
-        top: 50%;
-        transform: translate(-25px, -25px);
-        animation: a 2s forwards;
+        animation: progressVisibility 2s forwards;
       }
-      #spinner g {
-        transform-origin: center;
-        animation: b 2s linear infinite;
+      .progress-bar {
+        width: 276px;
+        height: 8px;
+        margin-top: 27px;
+        border-radius: 100px;
+        background: #d9d9d9;
       }
-      #spinner g circle {
-        stroke-linecap: round;
-        animation: c 1.5s ease-in-out infinite;
+      #progress {
+        width: 0;
+        height: 8px;
+        gap: 0px;
+        border-radius: 100px;
+        background: #4972bb;
+        animation: progressAnimation 300s ease;
```

Comment:
I have no science to back this up, but I assert that 90% case is that it'll load at least in 20 seconds - therefore it should rather load the progress bar to almost finished quite fast, and then lag in the last 10% in the progress. To serve well for most cases and then also give a feeling of "were nearly there" for the rare very slow cases.